rpc/grpc/server/services/blockresultservice/service.go

```diff
@@ -0,0 +1,61 @@
+package blockresultservice
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/cometbft/cometbft/libs/log"
+	sm "github.com/cometbft/cometbft/state"
+	"github.com/cometbft/cometbft/store"
+
+	brs "github.com/cometbft/cometbft/proto/tendermint/services/block_results/v1"
+)
+
+type blockResultsService struct {
+	stateStore sm.Store
+	blockStore *store.BlockStore
+	logger     log.Logger
+}
+
+// New creates a new CometBFT block results service server.
+func New(bs *store.BlockStore, ss sm.Store, logger log.Logger) brs.BlockResultsServiceServer {
+	return &blockResultsService{stateStore: ss, blockStore: bs, logger: logger}
```

Comment:
```suggestion
	return &blockResultsService{
		stateStore: ss,
		blockStore: bs,
		logger: logger.With("service", "BlockResultsService"),
	}
```

Helps with logging context.